homeassistant/helpers/frame.py

```diff
@@ -68,11 +71,13 @@ def report(
         _LOGGER.warning(msg, stack_info=True)
```

Comment:
I chose to leave this as `_LOGGER.warning` but maybe I should have changed it also?
```suggestion
        _LOGGER.log(level, msg, stack_info=True)
```

Reply:
This should probably change as well